node/resolvers/catalog/index.ts

```diff
@@ -255,16 +361,36 @@ export const queries = {
         ) as any
       }
 
-      response.category = found && found.id as any
+      response.category = found && (found.id as any)
     }
 
     return response
   },
 }
 
+interface Brand {
+  id: string
+  name: string
+  isActive: boolean
+  title: string
+  metaTagDescription: string
+}
+
 interface Category {
   id: string
   name: string
   url: string
   children: Category[]
+  hasChildren: boolean
+  Title: string
```

Comment:
Make a field resolver to be camelCase

```ts
title: string
metaTagDescription: string
```

Reply:
This is the TypeScript interface; the GraphQL API already deals with that [here](https://github.com/vtex-apps/store-graphql/blob/4923df375fe887f5f9fbfb31a551f2e9f23f1f59/node/resolvers/catalog/category.ts#L35) and [here](https://github.com/vtex-apps/store-graphql/blob/4923df375fe887f5f9fbfb31a551f2e9f23f1f59/node/resolvers/catalog/category.ts#L55).